pkg/command/cert.go

```diff
@@ -287,6 +298,71 @@ func createCert(host, caCertPath, skipCertVerify, insecure string) (*configtypes
 	return cert, nil
 }
 
+func updateCert(updateCert *configtypes.Cert, caCertPath, skipCertVerify, insecure string) (*configtypes.Cert, error) {
+	if caCertPath != "" {
+		fileBytes, err := os.ReadFile(caCertPath)
+		if err != nil {
+			return nil, errors.Wrapf(err, "error reading CA certificate file %s", caCertPath)
+		}
+		updateCert.CACertData = base64.StdEncoding.EncodeToString(fileBytes)
+		// Reset skip cert verify is ca cert is specified
+		updateCert.SkipCertVerify = "false"
```

Comment:
nit: is -> if

Reply:
Updated the comment

Reply:
I don't see the comments fixed?  Is there an update that wasn't pushed?